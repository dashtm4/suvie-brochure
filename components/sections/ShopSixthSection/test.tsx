import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ShopSixthSection } from '.'

describe('<ShopSixthSection />', () => {
  it('should renders correct <ShopSixthSection />', () => {
    const wrapper = shallow(<ShopSixthSection />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
