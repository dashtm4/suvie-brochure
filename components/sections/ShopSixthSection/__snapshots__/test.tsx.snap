// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShopSixthSection /> should renders correct <ShopSixthSection /> 1`] = `
<section
  className="bg-quaternary pt-12 pb-24"
>
  <Container>
    <SectionTitle
      className="mb-16 md:mb-8"
    >
      Reviews
    </SectionTitle>
    <YotpoReviews />
  </Container>
</section>
`;
